JS-basics/js-basics_dark-mode/js/index.js

```diff
@@ -1,3 +1,21 @@
 console.clear();
 
 const bodyElement = document.querySelector('[data-js="body"]');
+const dark = document.querySelector('[data-js="dark-mode-button"]');
+const light = document.querySelector('[data-js="light-mode-button"]');
+const toggle = document.querySelector('[data-js="toggle-button"]');
+
+dark.addEventListener("click", () => {
+  bodyElement.classList.add("button--light");
+  bodyElement.classList.add("button--dark");
+});
+
+light.addEventListener("click", () => {
+  bodyElement.classList.remove("button--dark");
+  bodyElement.classList.add("button--light");
```

Comment:
light mode button only removes dark features from the html code. you dont need ot add light  mode, default is already light.